scripts/run_regtest.sh

```
@@ -64,34 +68,37 @@ sleep 1
         --log-filters INFO \
         --network=regtest \
         --daemon-rpc-addr=${DOCKER_IP}:${BITCOIND_RPC_PORT} \
         --daemon-p2p-addr=${DOCKER_IP}:${BITCOIND_PORT} \
-        --electrum-rpc-addr=127.0.0.1:${ELECTRS_RPC_PORT1} \
+        --electrum-rpc-addr=0.0.0.0:${ELECTRS_RPC_PORT1} \
+        --monitoring-addr=0.0.0.0:${ELECTRS_MONITORING_PORT1} \
         --daemon-dir=/bitcoin/.bitcoin \
         --db-dir=/data &
 
 docker run \
        -u $(id -u $USER) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)